
Creating model.....
Creating training dataset...
Creating valid dataset...
train_loader length: 94
valid_loader length: 32





Epoch [0][93/94] | Train_loss 0.10016 | Train_accuracy_value 0.01299 | Train_accuracy_average 0.01748
Traceback (most recent call last):
  File "main.py", line 198, in <module>
    train(args)
  File "main.py", line 155, in train
    .format(epoch, iter, len(valid_loader), loss=losses, acc=acc), end='')
NameError: name 'valid_loader' is not defined